torch/_dynamo/backends/debugging.py

```diff
@@ -30,6 +31,53 @@ def eager(gm, fake_tensor_inputs, **kwargs):
         log.warning("eager backend ignoring extra kwargs %s", kwargs)
     return gm.forward
 
+@register_backend
+def eager_warmup_conditional_nodes(gm, fake_tensor_inputs):
+    s = torch.cuda.Stream()
+    with ControlFlowOpWarmupDispatchMode(s):
+        gm.forward(*fake_tensor_inputs)
+    return gm.forward
+
+class ControlFlowOpWarmupDispatchMode(TorchDispatchMode):
+    def __init__(self, stream):
```

Comment:
nit: any reason to pass the stream in here instead of instantiate inside ? it's not used anywhere else